content/en/blog/2021/announcing-notation-alpha1.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Announcing notation alpha 1
+spelling: cSpell:ignore
+author:  "[Steve Lasker](https://github.com/stevelasker), Microsoft Azure"
+date: 2021-10-20
+draft: false
+---
+
+We are happy to announce the [alpha 1 release][notation-release] of the [Notary v2 project][notary-v2].
+Notary v2 enables signing of all artifacts (Container Images, Software Bill of Materials, Scan Results) stored in an [OCI Distribution][oci-distribution] based registries.
```

Comment:
Should we refer to "Notation" here, rather than "Notary V2?"
```suggestion
We are happy to announce the [alpha 1 release][notation-release] of the [Notation project][notary-v2].
Notation (formerly named Notary V2) enables signing of all artifacts (Container Images, Software Bill of Materials, Scan Results) stored in [OCI Distribution][oci-distribution] based registries.
```

Reply:
The project is Notary, the version is 2, and there are specific components with discrete names.
Within the Notary Project, there's
- notation - a cli for sign, push, validate
- notation-go-lib for implementing the sign, push, validate capabilities in other clients
- tuf for enabling notary v2 signing with tuf metadata